app/src/main/java/ml/docilealligator/infinityforreddit/activities/ViewPostDetailActivity.java

```diff
@@ -829,43 +890,55 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
     @Override
     protected void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
```

Comment:
`onActivityResult` is deprecated and should be replaced with `[ActivityResultCallback](https://developer.android.com/reference/androidx/activity/result/ActivityResultCallback)`. IIRC these callbacks would match extracted functions